### PR TITLE
Add support for alerting

### PR DESCRIPTION
**Pull request by CharlesJUDITH**
This review will: 
- support of the new alerting feature in Grafana
- update the OS used in kitchen CI
- update the chefspec tests
- Update Grafana release link
- Update Grafana version

### CODE REVIEW

**Comment by iksaif** [comment]
Can you add a spec test maybe ? (there should be a few examples)

**Comment by CharlesJUDITH** [comment]
Sure I'll do it and push it there..

File: .kitchen.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 driver:
```
Comment (iksaif) [inline .kitchen.yml:2]: please change that in a separate commit, else it will be a pain to rebase

File: attributes/default.rb [hunk 2]
```diff
@@ -17,11 +17,11 @@
 
 default['grafana']['manage_install'] = true
 default['grafana']['install_type'] = 'file' # file | package | source
-default['grafana']['version'] = '2.1.2'
```
Comment (iksaif) [inline attributes/default.rb:20]: please change in a yet another commit, same with urls and checksums

File: spec/unit/default_spec.rb [hunk 3]
```diff
@@ -167,7 +177,7 @@
         context 'with LDAP authentication enabled' do
           let(:chef_run) do
             ChefSpec::SoloRunner.new(chef_solo_opts) do |node|
-              node.set['grafana']['ini']['auth.ldap']['enabled']['value'] = true
```
Comment (iksaif) [inline spec/unit/default_spec.rb:170]: This too in another commit

**Review by iksaif — CHANGES_REQUESTED** [review]
Please try to separate unrelated changes in multiple commits:
* One to bump the versions
* One for node.normal
* One that is *only* for alerting

Else it will be impossible to send those changes upstream

Thanks

File: .kitchen.yml (hunk 1 above)
Comment (CharlesJUDITH) [reply .kitchen.yml:2]: done

File: attributes/default.rb (hunk 2 above)
Comment (CharlesJUDITH) [reply attributes/default.rb:20]: done

File: spec/unit/default_spec.rb (hunk 3 above)
Comment (CharlesJUDITH) [reply spec/unit/default_spec.rb:170]: done

**Review by iksaif — APPROVED** [review]

**Comment by iksaif** [comment]
cool

**Pull request merged**